projects/demo/src/modules/components/button/examples/1/index.less

```diff
@@ -17,3 +17,8 @@
         animation: rotating 1.5s linear infinite;
     }
 }
+
+button:hover,
+a:hover {
+    outline: 2px solid var(--tui-accent-hover);
```

Comment:
I think this might confuse people, let's not add it just for testing.

Reply:
fixed